modules/mobile/app/services/mobile/v0/sessions/token_fetcher.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'capybara'
+require 'pry'
+require 'yaml'
+
+class TokenFetcher
+  TOKEN_URL = 'https://va-mobile-cutter.herokuapp.com'
+  USERS_FILE = ENV['STAGING_TEST_USERS']
+
+  attr_reader :token
+
+  def initialize(user_name)
+    set_user(user_name)
+    Capybara.default_max_wait_time = 10
+    @session = Capybara::Session.new(:selenium)
+  end
+
+  def fetch_token
+    @session.visit TOKEN_URL
+    @session.click_link('Please Login')
+    sleep 3 # this sleep appears to be necessary for dealing with the redirect
```

Comment:
instead of having a sleep is it possible to hold until the 'Sign in with ID.me' or some other button from the next page is detected?